readme.md

```diff
@@ -23,3 +27,26 @@
 5) Run your tests
 
 To enable debugging, set `BUILDKITE_ANALYTICS_DEBUG_ENABLED=true`
+
+### Jest Collector Roadmap
+
+#### DONE
+
+- [x] Baseline metric collection, such as test duration and pass/fail states
+- [x] Error messaging
+
+#### TODO
+
+- [ ] Support for deep-dives into spans via records of time spent in HTTP requests or running SQL queries
+- [ ] Support for annotations
+- [ ] Web socket support
```

Comment:
```suggestion
- [ ] Tests
- [ ] Sending through `failure_expanded`
- [ ] HTTP tracing
- [ ] SQL tracing
- [ ] Annotations
- [ ] Live stream results via web sockets
- [ ] Additional test framework support (Cypress , Jasmine, AVA, Mocha, etc)
```

Reply:
I lost your edits when I quickly in-browser resolved some conflicts yesterday @toolmantim . Have a look at my latest commit, should have them back now.